.config/GitVersion.yml

```diff
@@ -0,0 +1,5 @@
+branches: {}
+ignore:
+  sha: []
+merge-message-formats: {}
+mode: ContinuousDeployment
```

Comment:
Let's keep the pull request to just adding support for .NET 8. The diff is 2600+ LOCs.